javalib/src/main/scala/java/net/PlainSocketImpl.scala

```diff
@@ -74,14 +74,15 @@ private[net] class PlainSocketImpl extends SocketImpl {
   }
 
   override def bind(addr: InetAddress, port: Int): Unit = {
-    val hints = stackalloc[addrinfo]
-    val ret   = stackalloc[Ptr[addrinfo]]
-    string.memset(hints.asInstanceOf[Ptr[Byte]], 0, sizeof[addrinfo])
```

Comment:
Since `stackalloc` does call [`memset`](https://github.com/scala-native/scala-native/blob/master/nativelib/src/main/scala/scala/scalanative/unsafe/package.scala#L278), couldn't you have just removed this since `stackalloc` doesn't require a `Zone`? 

I could be missing something unless there is another problem being solved here.

Reply:
Good catch, thank you.   

The astounding and alien knowledge that stackalloc[] clears came to me only a few hours ago and I have
not yet absorbed it.  Still running on old tapes...

Fixed now.

Reply:
Thanks, `stackalloc` is the basic standard C on the stack but here is like special magic and zeroed.